.github/workflows/test.yaml

```
@@ -0,0 +1,81 @@
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Dagger Version
+        uses: sagikazarmark/dagger-version-action@v0.0.1
+
+      - name: Run Vulnerability Check
+        uses: dagger/dagger-for-github@v7
+        with:
+          version: ${{ steps.dagger_version.outputs.version }}
```

actionlint (1.7.4)

property "dagger_version" is not defined in object type {}

(expression)